src/explore-education-statistics-common/src/modules/table-tool/components/PublicationForm.tsx

```diff
@@ -142,7 +142,7 @@ const PublicationForm = ({
                   {searchTerm && publications.length > 0 && (
                     <p>
                       <a
-                        href={`${formId}-publications`}
+                        href={`#${formId}-publications`}
```

Comment:
Looks like the id it's linking to is wrong so it doesn't take you to the publications list. It's `publicationForm-publicationForm-publications`  - probably want to find what's adding the second `publicationForm` and remove it (checking that doesn't break anything else).

Reply:
hmm, interesting. afaik it looks like `FormFieldRadioGroup` was inheriting the ID from the `Form` component. I've removed the `${formId}` from the id and replaced it with `id="publications"` which seems to fix the duplication & link